src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/utils/DateUtils.kt

```diff
@@ -0,0 +1,8 @@
+package uk.gov.justice.digital.hmpps.hmppsworkload.utils
+
+import java.time.format.DateTimeFormatter
+
+object DateUtils {
+  @JvmStatic
+  val notifyDateFormat = DateTimeFormatter.ofPattern("d MMMM yyyy")
+}
```

Comment:
I am curious to know why this was chosen over a companion object or even  a const in EmailNotificationService - I have not seen @JvmStatic before